src/document.c

```diff
@@ -825,7 +825,8 @@ GeanyDocument *document_new_file_if_non_open(void)
 }
 
 
-/**
+/** @girskip
```

Comment:
Couldn't we just put `@girskip` on the original `document_new_file` function comments to have it ignored rather than upsetting the API to work around GI tool quirks?